### PR TITLE
Add mutation request sign in link

### PR DESCRIPTION
**Pull request by medson10**

### CODE REVIEW

**Review by georgelima — APPROVED** [review]

File: executable/SummarizeApp.re [hunk 1]
```diff
@@ -171,62 +127,19 @@ let message =
...
-            ~typ=non_null(message),
-            ~args=Arg.[],
-            ~resolve=(info, p: edge(User.Message.t)) =>
-            p.node
-          ),
-        ]
-      )
-    )
-  );
+module MessageConfig = {
```
Comment (fakenickels) [inline executable/SummarizeApp.re:130]:
```suggestion
module MessageConnectionConfig = {
```

File: executable/SummarizeApp.re [hunk 2]
```diff
@@ -272,17 +185,8 @@ let notification =
...
-            ~args=
-              Arg.[
-                arg("first", ~typ=float),
-                arg("after", ~typ=string),
-                arg("last", ~typ=int),
-                arg("before", ~typ=string),
-              ],
-            ~resolve=(_info, p, first, after, last, before) =>
+          MessageConnection.connectionResolver(
+            "messages", (_info, p, first, after, last, before) =>
```
Comment (fakenickels) [inline executable/SummarizeApp.re:189]: thinking why aren't we calling this `notifications`? 🤔

File: executable/SummarizeApp.re [hunk 3]
```diff
@@ -336,10 +240,14 @@ let user =
           field(
             "notifications",
             ~args=Arg.[arg("filter", ~typ=notificationsArg)],
-            ~typ=list(notification),
+            ~typ=list(non_null(notification)),
             ~resolve=(_info, p, filter) =>
             None
           ),
+          Organization.Connection.connectionResolver(
```
Comment (fakenickels) [inline executable/SummarizeApp.re:247]: nice exactly this

File: graphql-helpers/Connection.re [hunk 4]
```diff
@@ -0,0 +1,138 @@
+open Graphql_lwt;
+
+module type Config = {
+  type nodeType;
+  type context;
+  let nodeResolver: Schema.typ(context, option(nodeType));
+  let nodeName: string;
+};
+
+module Create = (Config: Config) => {
```
Comment (fakenickels) [inline graphql-helpers/Connection.re:10]:
```suggestion
module Make = (Config: Config) => {
```
OCaml feels

File: executable/SummarizeApp.re (hunk 2 above)
Comment (medson10) [reply executable/SummarizeApp.re:189]: we have notifications and inside of it messages

File: graphql-helpers/Connection.re [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+    let resolver =
+      Schema.(
+        obj("PageInfo", ~doc="Connections page info", ~fields=_ =>
+          [
+            field(
+              "startCursor",
+              ~doc="",
+              ~typ=string,
+              ~args=Arg.[],
+              ~resolve=(info, p) =>
```
Comment (fakenickels) [inline graphql-helpers/Connection.re:29]:
```suggestion
              ~resolve=(_info, p) =>
```
let's make some effort to clean unnecessary warnings, tag the unused variables

File: graphql-helpers/Connection.re [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+            ~args=Arg.[],
+            ~resolve=(info: Config.context, p: connection) =>
+            p.edges
+          ),
+          field(
+            "pageInfo",
+            ~doc="PageInfo of connection",
+            ~typ=non_null(PageInfo.resolver),
+            ~args=Arg.[],
+            ~resolve=(info: Config.context, p: connection) =>
```
Comment (fakenickels) [inline graphql-helpers/Connection.re:119]:
```suggestion
            ~resolve=(_info, p: connection) =>
```

File: graphql-helpers/Connection.re [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+  let connectionType =
+    Schema.(
+      obj(Config.nodeName ++ "Connection", ~fields=_ =>
+        [
+          field(
+            "edges",
+            ~doc="Edges of connection",
+            ~typ=list(non_null(edgeResolver)),
+            ~args=Arg.[],
+            ~resolve=(info: Config.context, p: connection) =>
```
Comment (fakenickels) [inline graphql-helpers/Connection.re:111]:
```suggestion
            ~resolve=(_info, p: connection) =>
```

File: graphql-helpers/Connection.re [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+            ~doc="PageInfo of connection",
+            ~typ=non_null(PageInfo.resolver),
+            ~args=Arg.[],
+            ~resolve=(info: Config.context, p: connection) =>
+            p.pageInfo
+          ),
+        ]
+      )
+    );
+  let connectionResolver = (name, cb) =>
```
Comment (fakenickels) [inline graphql-helpers/Connection.re]:
```suggestion
  let connectionResolver = (~args=[], name, cb) =>
```

File: graphql-helpers/Connection.re (hunk 4 above)
Comment (medson10) [reply graphql-helpers/Connection.re:10]: hhahah okay nice one

File: graphql-helpers/Connection.re [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+    Schema.field(
+      name,
+      ~typ=connectionType,
+      ~args=
+        Schema.Arg.[
+          arg("first", ~typ=float),
```
Comment (fakenickels) [inline graphql-helpers/Connection.re:131]:
```suggestion
          ...args,
          arg("first", ~typ=float),
```

**Review by fakenickels — APPROVED** [review]

File: graphql-helpers/Connection.re [hunk 10]
```diff
@@ -122,16 +122,13 @@ module Create = (Config: Config) => {
         ]
       )
     );
-  let connectionResolver = (name, cb) =>
+  let connectionResolver = (~args=Schema.Arg.[], name, cb) =>
```
Comment (fakenickels) [inline graphql-helpers/Connection.re:125]: top

**Pull request merged**